src/framework/ui/internal/uiconfiguration.cpp

```diff
@@ -173,3 +179,58 @@ float UiConfiguration::physicalDotsPerInch() const
     }
     return 100;
 }
+
+QByteArray UiConfiguration::state(const std::string& objectName) const
```

Comment:
I think, it is better to rename this method to windowState(pageName) or pageState(pageName)